runtime-modules/content/src/nft/mod.rs

```diff
@@ -345,11 +345,10 @@ impl<T: Trait> Module<T> {
                 _ => (),
             };
 
-            // Should always be Some(_) at this stage, because of previously made check.
-            if let Some(creator_account_id) = Self::channel_by_id(in_channel).reward_account {
-                // Deposit royalty into creator account
-                T::Currency::deposit_creating(&creator_account_id, royalty);
-            }
+            // Deposit royalty into creator account
```

Comment:
This fragment is getting repeated a lot, probably will only increase, perhaps make it into a method?

Reply:
Ok, I had to unify the balance type used: removed `CurrencyOf<T>` in favor of `BalanceOf<T>` in order to properly reuse the new method `deposit_to_creator`

Reply:
Is that really needed? its always desirable to use trait rather than implementation, e.g. for testing purposes. We do this sort of payment many other places, why does that work fine?